core/core-impl/src/test/java/io/fabric8/launcher/core/impl/preparers/ChangeArquillianConfigurationPreparerIT.java

```diff
@@ -88,11 +88,11 @@ public void shouldUpdateArquillianConfiguration() throws IOException, ExecutionE
       // then
       assertThat(
          contentOf(Paths.get(rootPath.toString(), "src/test/resources/arquillian.xml").toFile())
-            .contains("<property name=\"app.name\">my-artifact-id</property>"));
```

Comment:
That works, but I wonder if it isn't easier to do `assertThat(contentOf(...)).contains(...)`?

Reply:
I think this is only easier way with which we can check content of file

Reply:
The error message is more descriptive using the suggestion above, but I am totally fine if you want to keep like that

Reply:
If I used `assertThat(contentOf(...)).contains(...)` on single line then user has to scroll to check entire line, this is why I split it into multiple line